packages/restapi/src/lib/chat/approveRequest.ts

```diff
@@ -34,38 +34,44 @@ export const approve = async (
     signer = null,
     senderAddress,
     env = Constants.ENV.PROD,
-    // pgpPrivateKey = null,
+    pgpPrivateKey = null,
   } = options || {};
 
   if(account == null && signer == null) {
     throw new Error(`At least one from account or signer is necessary!`);
   }
 
+  if(!pgpPrivateKey) {
+    console.warn("Please note that if you don't pass the pgpPrivateKey parameter, a wallet popup will appear every time the approveRequest endpoint is called. We strongly recommend passing this parameter, and it will become mandatory in future versions of the API.")
```

Comment:
This warning comes for new user too. I think this should only come fpr old users so we should shift it to getConnectedUserV2 method